services/motion/explore/explore.go

```diff
@@ -323,6 +330,10 @@ func (ms *explore) checkForObstacles(
 	plan motionplan.Plan,
 	obstaclePollingFrequencyHz float64,
 ) {
+	ms.logger.Debug("Current Plan")
```

Comment:
[note] Additional useful log line but can be removed if desired

Reply:
it's nice!

Reply:
@raybjork any strong feelings on removing this log?